src/lib/viewers/box3d/image360/Image360Loader.js

```diff
@@ -27,6 +27,12 @@ const VIEWERS = [
             'tga',
             'tif',
             'tiff',
+            'cr2',
```

Comment:
Do we support 360 images of these types?

Reply:
Not sure. What is image360? I added it here because all other image formats are also here. And for these raw images we serve jpg representations.

Reply:
@qliu-box, is there a conversion pipeline in place to create `3d` reps for these new file extensions? If not, we should probably remove them from this list.

Reply:
Have confirmed with Derrick to remove this change. It's not necessary to provide 3d reps for raw images